NEWS.md

```diff
@@ -59,13 +59,24 @@ default. To enable it the new admiral option `save_memory` has to be set to
 
 ## Breaking Changes
   
-- The following function arguments are entering the next phase of the deprecation process: (#2487)
+- The following function arguments are entering the next phase of the [deprecation process](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation): (#2487) (#2595)
 
-### Phase 1 (warning)
+**Phase 1 (message)**
+  
+  - `derive_param_extreme_record()` is deprecated and replaced by `derive_extreme_event()`
+  - `derive_var_dthcaus()` is deprecated and replaced by `derive_vars_extreme_event()`
+  - `date_source()` is deprecated and replaced by `event()`
+  - `dthcaus_source()` is deprecated and replaced by `event()`
+  - `derive_var_extreme_dt()` and `derive_var_extreme_dtm()` are deprecated and replaced by `derive_vars_extreme_event()`
+  - `get_summary_records()` is deprecated. Please use `derive_summary_records()` with the `dataset_add` argument and without the `dataset` argument.
+
+  
+**Phase 2 (warning)**
+
+**Phase 3 (error)**
```

Comment:
It should be indented by four spaces to look nice.
```suggestion
    **Phase 1 (message)**
  
    - `derive_param_extreme_record()` is deprecated and replaced by `derive_extreme_event()`
    - `derive_var_dthcaus()` is deprecated and replaced by `derive_vars_extreme_event()`
    - `date_source()` is deprecated and replaced by `event()`
    - `dthcaus_source()` is deprecated and replaced by `event()`
    - `derive_var_extreme_dt()` and `derive_var_extreme_dtm()` are deprecated and replaced by `derive_vars_extreme_event()`
    - `get_summary_records()` is deprecated. Please use `derive_summary_records()` with the `dataset_add` argument and without the `dataset` argument.
  
    **Phase 2 (warning)**

    **Phase 3 (error)**
```
Should we remove "Phase 2" and "Phase 3"?

Reply:
Indented.

I like to keep those sections as placeholders so we keep things identical between releases.  I put in some text to be explicit on why they are empty.